x-pack/plugins/ml/server/models/data_recognizer/modules/security_host/ml/high_count_events_for_a_host_name.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Security: Host - Looks for a sudden spike in host based traffic. This can be due to a range of security issues, such as a compromised system, DDoS attacks, malware infections, privilege escalation, or data exfiltration.",
+  "groups": ["security", "host"],
+  "analysis_config": {
+    "bucket_span": "3h",
+    "detectors": [
+      {
+        "detector_description": "Detects high count of host based events.",
```

Comment:
I know this format is consistent with other security module detectors, but purely on the basis for how this looks inside the ML anomalies table, you could drop `Detects` and the closing `.` here to leave the more concise `high count of host based events`:


<img width="874" alt="Screenshot 2024-10-10 at 16 52 54" src="https://github.com/user-attachments/assets/a08ec43a-6a2a-4092-8e66-1389fe3cf99c">


Edit would give:

<img width="1283" alt="Screenshot 2024-10-10 at 16 54 43" src="https://github.com/user-attachments/assets/ea0a4e20-cd80-4054-8498-3c13c0de1e1f">